ceremonies/src/tests.rs

```diff
@@ -1293,13 +1298,102 @@ fn register_with_reputation_works() {
 	});
 }
 
+#[test]
+fn double_registering_by_adversary_bootstrapper_fails() {
+	new_test_ext().execute_with(|| {
+		let cid = perform_bootstrapping_ceremony(None, 1);
+		let alice = AccountKeyring::Alice.pair();
+		assert_ok!(EncointerCeremonies::claim_rewards(
+			RuntimeOrigin::signed(account_id(&alice)),
+			cid,
+			None
+		));
+		let cindex = EncointerScheduler::current_ceremony_index();
+		assert_eq!(cindex, 2);
+		let alice = AccountKeyring::Alice.pair();
+		// a non-bootstrapper newbie account controlled by adversary bootstrapper
+		let zoran = sr25519::Pair::from_seed_slice(&[9u8; 32]).unwrap();
+		// bootstrapper illegaly uses valid reputation to register fresh account as reputable
+		let proof = prove_attendance(account_id(&zoran), cid, cindex - 1, &alice);
+		assert_err!(
+			register(account_id(&zoran), cid, Some(proof)),
+			Error::<TestRuntime>::BootstrapperReputationIsUntransferrable
+		);
+		// now Alice abuses her bootstrapper privilege to try to register herself without proof
+		assert_ok!(register(account_id(&alice), cid, None));
+	});
+}
+
+#[test]
+fn register_as_bootstrapper_with_any_kind_of_reputation_after_unregister_works() {
```

Comment:
this test reproduces the problem with non-refunded reputation for bootstrappers in #367